### ISSUE TITLE
[Accessibility] Selected comment button colour does not contrast enough

### ISSUE THREAD

**Issue by pmberry** [openstreetmap/iD#11948]
### Description

1. When using the keyboard to navigate adding a comment to a note, when tabbing to the (Close and Comment) and (Comment) buttons, the highlighted one does not contrast enough against the un-highlighted one. This is a subjective issue which won't affect everyone but general web design guidelines recommend that highlight colours be as contrasting as possible to clearly show the current selection. Presently it's a slightly bolder shade of blue when, for full accessibility, it should be a different colour entirely, or perhaps with a border instead, for maximum effect. WAVE (a web accessibility evaluation tool) flags it as low contrast.
2. Further, the selected button is _lighter_ when in edit mode but _darker_ when in normal mode (albeit with a border).
3. Still further, the wording is different: (Close and Comment) in edit mode but (Comment & Resolve) in normal mode.

### Screenshots

Edit mode:
<img width="386" height="83" alt="Image" src="https://github.com/user-attachments/assets/3b978049-dac1-4d38-8472-907cc80a6eae" />

Normal mode:
<img width="330" height="162" alt="Image" src="https://github.com/user-attachments/assets/2506e358-5ad1-4fed-9a03-2c27e69e944f" />

**Comment by Shyam-123pandey** [comment]
Good morning Sir, 
I was able to reproduce the contrast difference between the selected and unselected buttons in both edit mode and normal mode. I agree that the visual distinction is quite subtle, especially when navigating via keyboard.
I’ll look into how the focus/active styles are currently handled in these states and see what improvements can be made to increase contrast and keep the behavior consistent across modes.
If there are any existing accessibility guidelines or preferred design direction in the project that I should follow, please let me know.

**Comment by dheeraj12347** [comment]
Hi, I noticed the contrast difference between the selected and unselected comment buttons when navigating via keyboard. I’d like to work on improving the visual distinction so the selected state is clearer and more accessible. Please let me know if it’s okay for me to take this issue.

**Comment by Shyam-123pandey** [comment]
I already working for this issue. thanks
